Makefile.am

```diff
@@ -35,8 +35,10 @@ AM_DISTCHECK_CONFIGURE_FLAGS=--without-systemdsystemunitdir
 
 nodist_systemdsystemunit_DATA = \
 	speech-dispatcherd.service
+nodist_systemduserunit_DATA = \
+	speech-dispatcher.service speech-dispatcher.socket
```

Comment:
Please surround this with `if HAVE_SYSTEMD`, to fix build on systems without systemd